test.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TEST</title>
+</head>
+<body>
+    
+    <h1> HACKERTOBERFEST</h1>
+
+    <h2> This is used for previewing output in console</h2>
+    <!-- <script src = "arithmeticOperators.js"></script> -->
+    <script src = "arrays.js"></script>
+
+
+</body>
+</html>
```

Comment:
This `test.html` file is not necessary. The repository is aimed at just solutions to the challenges.